plugins/source/github/resources/services/traffic/clones.go

```diff
@@ -0,0 +1,51 @@
+package traffic
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/google/go-github/v48/github"
+)
+
+func Clones() *schema.Table {
+	return &schema.Table{
+		Name:        "github_traffic_clones",
+		Description: "https://docs.github.com/en/rest/metrics/traffic?apiVersion=2022-11-28#get-repository-clones",
+		Resolver:    fetchClones,
+		Multiplex:   client.OrgRepositoryMultiplex,
+		Transform:   transformers.TransformWithStruct(&github.TrafficClones{}, client.SharedTransformers()...),
+		Columns: []schema.Column{
+			{
```

Comment:
let's wait till https://github.com/cloudquery/cloudquery/pull/8070 is merged and adopt the same definition approach?
(`client.OrgColumn` + maybe we need to move repo ID to client pkg as well)